docs/Installation/Installer-script.md

```diff
@@ -18,7 +18,7 @@ With `--config` you specify the path to the system configuration file, typically
 The script takes following commands:
 - [cluster](#cluster)
 - [node](#node)
-- [resources](#resources)
+- [storage](#resources)
```

Comment:
why did you rename this to storage?

Reply:
@grimpy  I looked at help of installer, resource was under cluster and storage was listed under installer.